editor.planx.uk/src/pages/Teams/AddTeamButton.tsx

```diff
@@ -0,0 +1,157 @@
+    await createTeam(values);
+    navigation.navigate(`/${values.slug}`);
+  };
+
+  const [dialogOpen, setDialogOpen] = useState<boolean>(false);
+
+  return (
+    <Permission.IsPlatformAdmin>
+      <AddButton onClick={() => setDialogOpen(true)}>Add a new team</AddButton>
+      <Formik<CreateTeam>
+        initialValues={initialValues}
+        onSubmit={onSubmit}
+        validateOnBlur={false}
+        validateOnChange={false}
+        validationSchema={validationSchema}
+      >
+        {({
+          resetForm,
+          getFieldProps,
+          errors,
+          setFieldValue,
+          values,
+          isSubmitting,
+        }) => (
+          <Dialog
+            open={dialogOpen}
+            onClose={() => {
+              setDialogOpen(false);
+              resetForm();
+            }}
+            aria-labelledby="alert-dialog-title"
+            aria-describedby="alert-dialog-description"
+            fullWidth
+          >
+            <DialogTitle variant="h3" component="h1">
+              <Box sx={{ mt: 1, mb: 4 }}>
+                <Typography variant="h3" component="h2" id="dialog-heading">
+                  Add a new team
+                </Typography>
+              </Box>
+            </DialogTitle>
+            <Box>
+              <Form>
+                <DialogContent
+                  sx={{ gap: 2, display: "flex", flexDirection: "column" }}
+                >
+                  <InputLabel label="Team name" htmlFor="name">
+                    <Input
+                      {...getFieldProps("name")}
+                      id="name"
+                      type="text"
+                      onChange={(e) => {
+                        setFieldValue("name", e.target.value);
+                        setFieldValue("slug", slugify(e.target.value));
+                      }}
+                      errorMessage={errors.name}
+                    />
+                  </InputLabel>
+                  <InputLabel label="Editor URL" htmlFor="slug">
+                    <Input
+                      {...getFieldProps("slug")}
+                      disabled
+                      type="text"
+                      startAdornment={<URLPrefix />}
+                    />
+                  </InputLabel>
+                  <Switch
+                    name="isTrial"
+                    checked={values.settings.isTrial}
+                    onChange={() =>
+                      setFieldValue(
+                        "settings.isTrial",
+                        !values.settings.isTrial,
+                      )
+                    }
+                    label={"Trial account"}
+                  />
+                  <Typography variant="body2" mt={-2}>
+                    A trial account has limited access to PlanX functionality
+                    (e.g. turning services online). A team can be promoted to
+                    having full access via the settings panel.
```

Comment:
Nit: in the second sentence, maybe "Trial teams can be promoted..." rather than "A team..." for clarity ?